data/card_catalogue/unique-1.json

```diff
@@ -64,37 +82,36 @@
     "upgrade": "",
     "tier": "red",
     "details": {
-      "rules": "Choose a Blue or Green card in your hand. All cards in your hand become copies of that card.",
+      "rules": {
+        "active": "Choose a Blue or Green card in your hand. All cards in your hand become copies of that card.",
+        "passive": ""
+      },
       "flavour": "Fax is short for facsimile, meaning \"make similar\".",
       "image": "assets/core/card/illustration/Card Printer.png"
     }
   },
   {
-    "title": "Nullify Magic",
+    "title": "Somnomalum",
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Prevent a spell from being cast, or remove magic from target object.",
-      "flavour": "What are you gonna do about it robe-boy, hit me with that little stick?",
-      "image": "assets/core/card/illustration/Nullify Magic.png"
-    }
-  },
```

Comment:
Thinking the same thing as:

> "We don't have an official notion of a 'spell'. Given that, the effect of this card is too niche"
> -@DuncanUszkay1, https://github.com/elliottomlinson/rpcg/pull/78#discussion_r677100083

Removed Nullify Magic, further discussion to take place in:
- #108